### ISSUE TITLE
Merge commits fail checks on develop

### ISSUE THREAD

**Issue by amarjandu** [DataBiosphere/azul#2906]
There are several merge commits that have failed the commit checks on the develop branch.
https://github.com/DataBiosphere/azul/commits/develop

<img width="1006" alt="Screen Shot 2021-03-17 at 12 07 17 PM" src="https://user-images.githubusercontent.com/1947534/111524340-89288b80-8719-11eb-974b-a9b560003ea2.png">

**Comment by jessebrennan** [comment]
This was occurring for various reasons.

One failure was due to #2684. I cleared the build cache, re-ran it, and it was successful.

Another was due to #2148. This is solved by re-running the build, but I haven't done this yet.

**Comment by jessebrennan** [comment]
All builds are now passing.

**Comment by hannes-ucsc** [comment]
![image](https://user-images.githubusercontent.com/5143256/111840327-7482f980-88b9-11eb-806f-67e7912840eb.png)

**Comment by jessebrennan** [comment]
I checked my bash command history to try and piece together how the mystery build happened, and unfortunately I have no clue at this point. Somehow I pushed d6f44a22 to the remote of the feature branch, but nothing in my command history indicates this.

It looks like you [cannot remove status check from a specific commit](https://webapps.stackexchange.com/a/78518) (but it is possible to overwrite it if the red X is causing issues). If this happens again, I'll keep a closer eye and check my shell's log before it's gone.
